app/komponenter/tekstBlokk/tekstBlokk.tsx

```diff
@@ -0,0 +1,44 @@
+import { TypografiTyper } from '~/typer/typografi';
+import React from 'react';
+import { PortableText } from '@portabletext/react';
+import { TypografiWrapper } from '~/utils/typografiWrapper';
+import css from './veilederhilsen.module.css';
+
+interface TekstBlokkProps {
+  tekstblokk: any | undefined;
+  valgBlock: string;
```

Comment:
Hva er valgBlock? Slik jeg ser det er det valgt språk? I så fall ville jeg kanskje renamet den så den beskriver at det er valgt språk og endret den til en enum i stedet for string

Reply:
valgBlock var bare å velge blokker for språk så helt enig og byttet navn her [2343bbb](https://github.com/bekk/nav-familie-endringsmelding/pull/13/commits/2343bbb3576b2b40055fcf696c663f31d3e6c282)